apps/epic-react/src/pages/workshops/[module]/[lesson]/index.tsx

```diff
@@ -48,8 +53,9 @@ export const getStaticProps: GetStaticProps = async (context) => {
 
 export const getStaticPaths: GetStaticPaths = async (context) => {
   const tutorials = await getAllWorkshops()
+  const bonuses = await getAllBonuses()
 
-  const paths = tutorials.flatMap((tutorial: any) => {
+  const paths = [...tutorials, ...bonuses].flatMap((tutorial: any) => {
```

Comment:
decided to merge `workshops` (it was "tutorials" here before, I didn't change) and `bonuses` here